mail_restrict_follower_selection/models/mail_followers.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018 Creu Blanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class MailFollowers(models.Model):
+    _inherit = 'mail.followers'
+
+    @api.model
+    def _add_follower_command(self, res_model, res_ids, partner_data,
+                              channel_data, force=True):
+        partners = self.env['res.partner'].search(
+            [('id', 'in', list(partner_data))] +
+            safe_eval(self.env['ir.config_parameter'].get_param(
```

Comment:
This should use the same scheme than the other domain, allowing to put a domain per model. See:

https://github.com/OCA/social/blob/e640e21bcc3f35a9b45ae97843f0540b505bcea0/mail_restrict_follower_selection/models/mail_wizard_invite.py#L30

Reply:
Good point! I have applied your suggestion